app/service/rest.py

```diff
@@ -793,8 +773,7 @@ def create_one_off_notification(service_id):
 
 @service_blueprint.route("/<uuid:service_id>/send-pdf-letter", methods=["POST"])
 def create_pdf_letter(service_id):
-    resp = send_pdf_letter_notification(service_id, request.get_json())
-    return jsonify(resp), 201
+    pass
```

Comment:
Are these routes reachable by end-users? If so, what kind of response is received after this change?

Reply:
I think they are currently in admin, though not visible. will remove them all in another pr